switools/swisigningservice.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3.7
+import os
+import sys
+from base64 import b64encode
+
+# Just a simple implementation of a signing service for testing purposes.
+# This signing service does not contact any signing server, it just uses a local 
+# signing key and uses that to sign the given sha256. Eos signatures are over the
+# sha256sum of the image with a 4k key size, we just handle that case.
+# The swi-signature script does the sha256 on the image and passes it to this script
+# (if the given options don't provide a signer's key, otherwise no need to call this
+# script). The swi-signature also passes the filename were the resulting signature
+# should be placed in.
+# This script assumes the signer's key to be in /etc/swi-signing-devCA/signing.key
+# (and does not bother to verify if it exists, this is just test code).
+
+def main():
+
+   if len( sys.argv ) != 3:
+      print( "Error" )
+      print( "usage: %s <sha256-string> <file-to-hold-signed-sha256-string>" % sys.argv[0] )
+      sys.exit( -1 )
+   
+   digest = sys.argv[1]
+   resultFile = sys.argv[2]
+   
```

Comment:
```suggestion
   keyFile = '/etc/swi-signing-devCA/signing.key'
```
or even better, make it an argument to the script, maybe with this as the default?

Reply:
We call swi-signing-service when we don't have a key in the first place, so cannot pass it as argument, that would be sort of senseless... Will change the default later ("devCA" not a good customer name...). And also pass it via environment for tests (where a key pair is created in /tmp/mytest).

Reply:
Maybe I'm missing something here, but this script _requires_ the keyfile to exist, so why can't it be passed as an argument?